app/src/main/res/layout/activity_create_key_gear_item.xml

```diff
@@ -63,12 +64,54 @@
         android:layout_marginStart="@dimen/base_margin_quadruple"
         android:layout_marginEnd="@dimen/base_margin_quadruple"
         android:layout_marginBottom="@dimen/base_margin_quadruple"
-        android:backgroundTint="@color/purple"
+        android:backgroundTint="@color/link_purple"
         android:text="@string/KEY_GEAR_ADD_ITEM_SAVE_BUTTON"
         android:textColor="@color/white"
         android:translationY="140dp"
         android:visibility="gone"
         tools:translationY="0dp"
         tools:visibility="visible" />
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/postCreate"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/link_purple"
+        android:translationZ="@dimen/post_create_z_translation"
+        android:visibility="gone">
+
+        <!-- TODO: Use animation assets when available -->
+        <ImageView
+            android:id="@+id/createdAnimation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/base_margin_octuple"
+            android:contentDescription="@null"
+            android:visibility="gone"
+            app:layout_constraintBottom_toTopOf="@id/createdLabel"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_chainStyle="packed"
+            tools:src="@drawable/ic_fire_green"
+            tools:visibility="visible" />
+
+        <!-- TODO: Use text key in code when available -->
```

Comment:
Looks like you have a text key

Reply:
You'd think so, but it's actually lies. We have the base text key, but no text keys for the values that must be interpolated into it.